packages/runtime/src/server/ServerBuilder.ts

```diff
@@ -33,8 +33,8 @@ export default class RuntimeBuilder
         const remoteBuilder = this.#remoteBuilder;
         const middlewareManager = await this.#buildMiddlewareManager(middleware);
         const healthManager = await this.#buildHealthManager(healthChecks);
-        const setUpScripts = setUp.map(filename => this.#makeSharedFilename(filename));
-        const tearDownScripts = tearDown.map(filename => this.#makeSharedFilename(filename));
+        const setUpScripts = setUp.map(filename => this.#makeCommonFilename(filename));
```

Comment:
Food for thought: set-up and tear-down scripts should be a part of the resources instead of common application files?